samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -85,6 +85,10 @@ This also means that, for example, when using four GPIOs, the pattern count cann
 
 If the number of switch-sample periods is greater than the number of stored switching patterns, then the radio loops back to the first pattern.
 
+The length of an antenna switch pattern is limited by :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.
+If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` add the Kconfig to a board configuration file.
```

Comment:
```suggestion
If the required length of the antenna switching pattern is greater than the default value of the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`, set this option to required value in a board configuration file.
```

Reply:
"The Kconfig" in this context is a little confusing to me

Reply:
I've changed it to:
```
If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`, set the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` with required value in a board configuration file.
```